### ISSUE TITLE
Power loss event? 

### ISSUE THREAD

**Issue by JohNan** [persandstrom/python-verisure#121]
Hi,

In the app I get notified when there's a power outage and it is also visible in the history tab in the app. Is it possible to have a binary sensor that flips to true if there's a power outage?

**Comment by persandstrom** [comment]
Hi,

I have not been able to catch notifications. You can view the history with the ```eventlog``` command and the power outage should be visible there. This module do not keep any state of your system so it will not be aware that something has changed, you will have to keep track of that in your application.

**Comment by JohNan** [comment]
I did try the eventlog but could not find the power loss event. Do you know in what category it might be?

**Comment by krambriw** [comment]
I don't think it is registered in the history. I know I had a power loss not long ago when they changed the meter but I cannot find any power loss event in history even if I look 1 year back

It's a pity they seem not willing to make an open api for notifications so that Verisure could be integrated with home automation more efficient. Having to poll is what we do but kind of stone age. Besides, it loads their servers much more, especially if you start to poll the history frequently

**Comment by persandstrom** [comment]
Is it possible to trigger the power loss event? I tried to unplug the main unit, but did not get any indication. Do I have to wait until the backup battery is depleted?

**Comment by krambriw** [comment]
There is a timeout but I think it is several, maybe at least ten minutes

**Comment by persandstrom** [comment]
I did a test and noticed that the power outage is visible in the overview. 
```
"installationErrorList": [
{
  "cid": ***,
  "zone": ***,
  "statusType": "ACPWR",
  "state": true,
  "date": ***,
  "name": ***,
  "deviceLabel": ***,
  "deviceType": "GATEWAY1",
  "notificationDeviceType": "GATEWAY",
  "area": ***
}
```
When the cable is plugged back in again the item in the installationErrorList is removed again.

I guess you can use the existence of this error as a "flag".
